app/views/guides/_writing_your_claim_statement.html.slim

```diff
@@ -48,4 +48,4 @@ article#writing_your_claim_statement
     li = t '.list_five.two'
 
   p.govuk-body
-    a.govuk-link href="#top" = t '.link_one'
+    a.govuk-link href="javascript:window.scrollTo({ top: 0 });" = t '.link_one'
```

Comment:
In general, inline javascript is ugly and we don't use it.  
What you should do is follow the pattern of other pages where the page has its own javascript module which you call from the slim file the end.  The special link would need some attribute to identify it

Also, I am confused because the ticket says the 'Back to form' link - but this is the 'back to top' link ?